### PR TITLE
refresh loadmore spinner (bug 1123869)

### PR DESCRIPTION
**Pull request by ngokevin**
- Don't replace button with spinner, just spinner the icon within the button
- Depends upon https://github.com/mozilla/marketplace-core-modules/commit/8c2f7dfcff09462d4ecf9920b3fdbcb897de8322 for builder changes to not hide btn on pagination
- Feed loadmore tests

- Remove some Persona code in main.js
- Resolve some TODOs in main.js

### CODE REVIEW

File: src/media/css/buttons-loadmore.styl [hunk 1]
```diff
@@ -90,8 +66,23 @@ $loadmore-desktop-width = 300px;
...
+    .grid-if-desktop .loadmore {
+        padding-top: 18px;
+    }
     [data-page-type~=homepage],
     [data-page-type~=app-list] {
         padding-bottom: 0;
     }
 }
+
+@-webkit-keyframes spin {
```
Comment (spasovski) [inline src/media/css/buttons-loadmore.styl]: These are already defined in `site.styl`

File: src/media/js/main.js [hunk 2]
```diff
@@ -1,4 +1,8 @@
-// Do this last- initialize the marketplace!
+/*
+    The main file that initializes the app.
+    Only put initialization code in here. Everything else should go into
+    separate and appropriate modules. This is not your diaper.
+*/
 console.log('Mozilla(R) FP-MKT (R) 1.0');
 console.log('   (C)Copyright Mozilla Corp 1998-2014');
```
Comment (spasovski) [inline src/media/js/main.js]: 2015?

File: src/media/js/main.js [hunk 3]
```diff
@@ -159,12 +148,11 @@ function(_) {
...
-        // Mark installed apps as such and look for a Marketplace update. If we
-        // are in a packaged app, wait for the iframe to be loaded, otherwise
-        // we are using the direct installer and we just need to wait for the
-        // normal loaded event.
+        // Mark installed apps, look for Marketplace update. If in packaged
+        // app, wait for the iframe to load. Else we use direct installer and
+        // just need to wait for normal loaded event.
         var is_packaged_app = window.location.protocol === 'app:';
-        var is_iframed_app = window.top !== window.self;  // Crude, but enough for our needs.
+        var is_iframed_app = window.top !== window.self;  //  Good enough.
```
Comment (spasovski) [inline src/media/js/main.js]: :lollipop:

File: src/media/js/main.js [hunk 4]
```diff
@@ -246,30 +236,20 @@ function(_) {
             z.body.removeClass('show-recommendations');
         }
 
-        siteConfig.promise.then(function () {
+        siteConfig.promise.then(function() {
```
Comment (spasovski) [inline src/media/js/main.js]: :+1:

File: src/media/css/buttons-loadmore.styl [hunk 5]
```diff
@@ -14,38 +14,23 @@ $loadmore-desktop-width = 300px;
...
-            margin: auto;
-            width: 32px;
-            height: 32px;
-            border-radius: 20px;
-            border: 4px solid #666;
-            border-color: #666 transparent;
-            animation(unquote('spin 1s linear infinite'));
+        &:before {
+            background: url('../img/icons/refresh-icon.svg');
+            bottom: 1px;
```
Comment (spasovski) [inline src/media/css/buttons-loadmore.styl]: This way of centering is awkward but I see why you did it. The result looks good.

**Comment by spasovski** [comment]
r+ looks great locally thanks for taking this on

**Comment by ngokevin** [comment]
thanks

**Pull request merged**